.github/release-drafter.yml

```diff
@@ -0,0 +1,52 @@
+name-template: 'v$NEXT_PATCH_VERSION 🌈'
```

Comment:
rule no 1, a 25 file PR with almost 5K line of code cannot be reviewed by a human being :) , best PR size as per research is ~400 lines of code.

Reply:
I get the point, will make sure to do small PRs from now 😓